python/nano/test/neural-compressor/pytorch/test_trainer_quantize.py

```diff
@@ -142,3 +158,26 @@ def test_trainer_quantize_inc_ptq_customized(self):
             assert loaded_qmodel
             out = loaded_qmodel(x)
             assert out.shape == torch.Size([256, 10])
+
+    def test_trainer_quantize_inc_ptq_customized_no_forward(self):
+        # Test if a Lightning Module not compiled by nano works
+        train_loader_iter = iter(self.train_loader)
+        x = next(train_loader_iter)[0]
+        trainer = Trainer(max_epochs=1)
+
+        qmodel = trainer.quantize(self.user_defined_pl_model_no_forward,
+                                  calib_dataloader=self.train_loader,
+                                  inference_method_name="predict")
+        assert qmodel
+        out = qmodel(x)
```

Comment:
better to support `out = qmodel.predict(x)`

Reply:
yeah, might be a good idea. I will first have a try and see if their is any block there.